projects/aws/eks-a-snow-admin-ami/provisioners/install_kubectl.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBECTL_URL="${KUBECTL_URL:-https://distro.eks.amazonaws.com/kubernetes-1-22/releases/4/artifacts/kubernetes/v1.22.6/bin/linux/amd64/kubectl}"
```

Comment:
Lets get this from the helper methods we use elsewhere for fetching eks-d binaries that we include in various images.

Reply:
https://github.com/aws/eks-anywhere-build-tooling/blob/main/build/lib/fetch_binaries.sh#L49

Reply:
Done